src/environmentd/src/http.rs

```diff
@@ -537,8 +541,7 @@ fn base_router(BaseRouterConfig { profiling }: BaseRouterConfig) -> Router {
             "/hierarchical-memory",
             routing::get(memory::handle_hierarchical_memory),
         )
-        .route("/static/*path", routing::get(root::handle_static))
-        .layer(DefaultBodyLimit::max(MAX_REQUEST_SIZE));
```

Comment:
Does moving this to the outer function still apply the limit to things within this funtion?

Reply:
The exact language is
> Note that the middleware is only applied to existing routes. So you have to first add your routes (and / or fallback) and then call `layer` afterwards. Additional routes added after `layer` is called will not have the middleware added.

My interpretation is that when you call `layer`, it is applied to all routes that have been added to the router before calling `layer`. So it would still apply to limit things within this functions.